tekton/publish.yaml

```diff
@@ -177,6 +183,8 @@ spec:
         IMAGE_WITHOUT_SHA_AND_TAG=${IMAGE_WITHOUT_SHA%%:*}
         IMAGE_WITH_SHA=${IMAGE_WITHOUT_SHA_AND_TAG}@${IMAGE##*@}
 
+        echo $IMAGE_WITH_SHA, >> $(results.IMAGES.path)
```

Comment:
this is probably a silly question but will a trailing comma be okay? im guessing yes and that the effort to remove it isn't worth it XD

Reply:
ps do you know if the crane cp commands below will also produce more unique shas?

Reply:
Yeah I think so - this is taken verbatim from chains own publish script: https://github.com/tektoncd/chains/blob/main/release/publish.yaml

Reply:
/cc @priyawadhwa

Reply:
> ps do you know if the crane cp commands below will also produce more unique shas?

Oo, this I don't know much about unfortunately.

Reply:
trailing comma should be fine!

& the`crane cp` command should result in the same digest!